### ISSUE TITLE
0.2.2 crashes when starting SPICE connection

### ISSUE THREAD

**Issue by bgilbert** [cmusatyalab/vmnetx-android#28]
Reliably triggers on Nexus 5/Android 6.0.1 and Pixel C/Android 7.0.  I believe this is the crash that `spice-marshaller-sigbus.patch` is meant to fix.

```
08-24 16:54:41.949 15478 15542 F libc    : Fatal signal 7 (SIGBUS), code 1, fault addr 0x9e47109e in tid 15542 (Thread-2630)
08-24 16:54:42.051  1439  1439 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
08-24 16:54:42.052  1439  1439 F DEBUG   : Build fingerprint: 'google/hammerhead/hammerhead:6.0.1/MOB30Y/3067468:user/release-keys'
08-24 16:54:42.052  1439  1439 F DEBUG   : Revision: '11'
08-24 16:54:42.052  1439  1439 F DEBUG   : ABI: 'arm'
08-24 16:54:42.053  1439  1439 F DEBUG   : pid: 15478, tid: 15542, name: Thread-2630  >>> org.olivearchive.vmnetx.android <<<
08-24 16:54:42.053  1439  1439 F DEBUG   : signal 7 (SIGBUS), code 1 (BUS_ADRALN), fault addr 0x9e47109e
08-24 16:54:42.105  1439  1439 F DEBUG   :     r0 9e47109e  r1 00000012  r2 f85c74aa  r3 000006b4
08-24 16:54:42.105  1439  1439 F DEBUG   :     r4 b3b34e48  r5 b3b34e60  r6 9e456338  r7 9e456470
08-24 16:54:42.105  1439  1439 F DEBUG   :     r8 9e456338  r9 00000000  sl 00000000  fp 9d8ebcc4
08-24 16:54:42.106  1439  1439 F DEBUG   :     ip 00000012  sp 9d67a820  lr 9e471094  pc 9d923438  cpsr 20000010
08-24 16:54:42.106  1439  1439 F DEBUG   : 
08-24 16:54:42.106  1439  1439 F DEBUG   : backtrace:
08-24 16:54:42.106  1439  1439 F DEBUG   :     #00 pc 0009a438  /data/app/org.olivearchive.vmnetx.android-2/lib/arm/libspice.so (spice_marshaller_add_uint64+24)
08-24 16:54:42.444  1439  1439 F DEBUG   : 
08-24 16:54:42.444  1439  1439 F DEBUG   : Tombstone written to: /data/tombstones/tombstone_04
08-24 16:54:42.444  1439  1439 E DEBUG   : AM write failed: Broken pipe
08-24 16:54:42.445  1919 15549 W ActivityManager:   Force finishing activity org.olivearchive.vmnetx.android/.RemoteCanvasActivity
```

**Comment by jaharkes** [comment]
the patch is applied, maybe the compiler is optimizing something away.

**Comment by jaharkes** [comment]
Should be fixed in 0.2.3 which is propagating to the Play store now.
